### ISSUE TITLE
[Server thread/CRITICAL]

### ISSUE THREAD

**Issue by DemonicNephalem** [onebone/EconomyS#446]
[Server thread/CRITICAL]: ErrorException: "yaml_parse(): reading error encountered during parsing: control characters are not allowed (line 1, column 1)" (EXCEPTION) in "src/pocketmine/utils/Config" at line 189

 [Server thread/INFO]: Disabling EconomyAPI v5.7.2
[12:00:25] [Server thread/CRITICAL]: Error: "Call to a member function setAll() on null" (EXCEPTION) in "plugins/EconomyAPI.phar/src/onebone/economyapi/provider/YamlProvider" at line 142

**Comment by onebone** [comment]
It seems like your Money.yml is not valid. Could you provide first 10 lines of Money.yml?

**Comment by DemonicNephalem** [comment]
Did not think you would reply that fast. Thank you but unfortunately im headed off to work so when i get off i will.

**Comment by DemonicNephalem** [comment]
@onebone yeah so i opened up the money.yml and theres nothing no text or anything. Its all gone.

**Comment by onebone** [comment]
Have you seen any error messages right before that error happens?

**Comment by DemonicNephalem** [comment]
No.

**Comment by DemonicNephalem** [comment]
@onebone after deleting and redownloading the pluggin i tried it again the first time it was the same issue with the error and money.yml being empty. However on the second time of deleting and redownloading the plugin for some reason there was no error when starting up the server but now the Money.yml just has- 

---
version: 2
money: [ ]
---

**Comment by onebone** [comment]
There must be an error message left on the server.log. Maybe it is worth checking it.
